pkg/feature/manifest.go

```diff
@@ -57,9 +57,7 @@ func (m *manifest) processTemplate(data interface{}) error {
 
 	f, err := os.Create(path)
 	if err != nil {
-		log.Error(err, "Failed to create file")
-
-		return err
+		return errors.Wrap(err, "failed to create file")
```

Comment:
I wonder how much we should be relying in that library behind `errors.Wrap`. It is no longer active since about 4years.

Alternatively: `return fmt.Errorf("failed to create file: %w", err)`, which is part of the std library.